groups/iam/module-iam-policies/test-data-generator-stack-terraform-destroy.tf

```diff
@@ -0,0 +1,53 @@
+data "aws_iam_policy_document" "test_data_generator_stack_terraform_destroy" {
+
+  statement {
+    sid       = "DestroyPolicyAllResources"
+    effect    = "Allow"
+    resources = ["*"]
+    actions = [
+      "ec2:DescribeImages",
+      "ecs:DeregisterTaskDefinition",
+    ]
+  }
+
+  statement {
+    sid    = "DestroyPolicyListedResources"
+    effect = "Allow"
+    resources = [
+      "arn:aws:ssm:eu-west-2:${var.aws_account_id}:parameter/test-data-generator-devops1/secret-mongo-url",
+      "arn:aws:iam::${var.aws_account_id}:role/test-data-generator-devops1-ecs-instance-role",
```

Comment:
These shouldn't be environment specific. I'd probably wildcard the environmental portion

Reply:
You thinking about wildcarding the region?
(and the same for any other lines that contain "_...something:**eu-west-2**:something...._")

Reply:
No I'm saying there shouldn't be any reference to `devops1`

Reply:
Ah yes, I see what you mean...
😖

Reply:
Done - environment has been wildcarded. Good shout there. 👍